langchain/src/schema/runnable/base.ts

```diff
@@ -1408,6 +1408,10 @@ export class RunnableWithFallbacks<RunInput, RunOutput> extends Runnable<
     this.fallbacks = fields.fallbacks;
   }
 
+  get getRunnable() {
```

Comment:
Call just `runnable`

Reply:
Can't since `runnable` is already defined above:
```typescript
  protected runnable: Runnable<RunInput, RunOutput>;
```